roles/fluentd_master/tasks/main.yml

```diff
@@ -39,8 +39,13 @@
     owner: 'td-agent'
     mode: 0444
 
-- name: "Pause before restarting td-agent and openshift-master, depending on the number of nodes."
-  pause: seconds={{ ( num_nodes|int < 3 ) | ternary(15, (num_nodes|int * 5)) }}
+- name: wait for etcd to start up
+  wait_for: port=4001 delay=10
+  when:  num_etcd|int == 0
+
+- name: wait for etcd peer to start up
+  wait_for: port=7001 delay=10
+  when:  num_etcd|int == 0
```

Comment:
I don't think num_etcd is being set anywhere outside of the bin/cluster invoked playbooks.

It might be better to set a variable to be passed into the role that is based off of the value of {{ groups.g_etcd_group | count }}. g_etcd_group is required to be set by any playbook invoking the common/openshift-cluster/config.yml playbook. The main reason I suggest setting an additional variable to be passed into the role is that it keeps variables that are set outside of the role from becoming required variables for the role, instead of the role staying somewhat self-contained.